Lib/test/test_capi/test_pyrun.py

```diff
@@ -0,0 +1,47 @@
+import unittest, os
+from test.support import import_helper
+
+# Skip this test if the _testcapi module isn't available.
+_testcapi = import_helper.import_module('_testcapi')
+
+NULL = None
+Py_single_input = _testcapi.Py_single_input
+Py_file_input = _testcapi.Py_file_input
+Py_eval_input = _testcapi.Py_eval_input
+
+class PyRunTest(unittest.TestCase):
+
+    # create file
+    def __init__(self, *args, **kwargs):
+        self.filename = "pyrun_fileexflags_sample.py"
+        with open(self.filename, 'w') as fp:
+            fp.write("import sysconfig\n\nconfig = sysconfig.get_config_vars()\n")
+            fp.close()
+            unittest.TestCase.__init__(self, *args, **kwargs)
```

Comment:
```suggestion
```

Reply:
The `open()` context manager will close the file, and calling `unittest.TestCase.__init__` is not needed (we use the `setUp`/`tearDown` API instead).